packages/core/src/lib/store/index.ts

```diff
@@ -79,7 +79,8 @@ class Store extends BaseProtocol implements IStore {
 
   constructor(libp2p: Libp2p, options?: ProtocolCreateOptions) {
     super(StoreCodec, libp2p.components);
-    this.pubsubTopics = this.initializePubsubTopic(options?.shardInfo);
+    this.pubsubTopics =
```

Comment:
It is duplicated - let's make `this.initializePubsubTopic(options.pubsubTopics, options.shardInfo)` or something.

Reply:
ok, I think I figured out a good approach: have `initializePubsubTopic` accept `ProtocolCreateOptions` and have the deprecated logic in there. When it comes time to remove named sharding, we'll just need to update the function instead of each instance where it's called. Updated in latest commit